scripts/update-version.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+async function runUpdate() {
+    const langiumPath = getPath('langium', true);
+    const langiumPackage = await fs.readJson(langiumPath);
+    const version = langiumPackage.version;
+    await Promise.all([
+        replaceAll('langium-cli', true, version),
```

Comment:
You have to add `langium` here as well otherwise the `langium-cli` devDependency is unchanged and then polluting the lock file with old versions.